tests/webview/integration/test_redirects_301.py

```diff
@@ -2,136 +2,77 @@
-def test_archive_is_still_reachable(archive_base_url, rex_base_url):
-    """REX still needs a way to fetch the content from Archive without being redirected
-    """
-    # GIVEN an archive URL for a book
-    url = f"{archive_base_url}/contents/f8zJz5tx@11.3:Pw-p-yeP@10/10-3-Phase-Transitions"
-
-    # WHEN making a request to Archive
-    response = requests.get(url)
-
-    # THEN we should NOT redirect to REX
-    for hist in response.history:
-        assert rex_base_url not in hist.headers["location"]
-
-
-@markers.rex
-@markers.test_case("C553080")
-@markers.nondestructive
-def test_redirecting_to_rex_from_within_webview(
-    webview_base_url, selenium, rex_base_url, rex_released_books
-):
-
-    # GIVEN the home page and REX released books list
-    for rex_book in rex_released_books:
-
-        home = Home(selenium, webview_base_url).open()
-
-        for cnx_book in home.featured_books.openstax_list:
-
-            # WHEN we click on a featured book
-            if cnx_book.cnx_id in rex_book:
-
-                book_title = cnx_book.title
-
-                title_link = home.driver.find_element_by_link_text(book_title)
-                title_link.click()
-
-                current_url = home.current_url
-
-                # THEN we redirect to REX
-                assert rex_base_url in current_url
-                break
-
-        else:
-            assert False, f"{rex_book} not a rex book"
-
-
-@markers.rex
-@markers.nondestructive
-def test_minimal_view_for_android_apps(webview_base_url, rex_base_url):
-    """All requests for REX books that come from the Android App
-    should continue to pass through to the cnx site (these requests
-    are indicated by the attachment of the query-string: `?minimal=true`)
-    https://github.com/openstax/cnx/issues/401
-    """
-    # GIVEN a cnx book url for which there is a REX-version
-    url = f"{webview_base_url}/contents/f8zJz5tx@11.3:Pw-p-yeP@10/10-3-Phase-Transitions"
-    response = requests.get(url)
-    assert rex_base_url in response.url
-
-    # WHEN we include the minimal view query in the request
-    minimal_view_url = f"{url}?minimal=true"
-    response = requests.get(minimal_view_url)
-
-    #  THEN we do not redirect to REX
-    assert rex_base_url not in response.url
-
-
-@markers.test_case("C553081")
+@markers.test_case("C568716")
 @markers.slow
 @markers.rex
 @markers.nondestructive
 def test_chemistry_2e_uris_redirect_to_rex(webview_base_url, rex_base_url, chemistry_2e_uri):
     # GIVEN a webview_base_url, rex_base_url and a chemistry_2e_uri
 
     # WHEN we go to a page based on the webview_base_url and uri
-    cnx_page_slug = chemistry_2e_uri.split("/")[-1]
     cnx_url = f"{webview_base_url}{chemistry_2e_uri}"
-    response = get_url(cnx_url)
+    response = requests.get(cnx_url, allow_redirects=False)
 
     # THEN we are redirected to rex
-    assert response.url.startswith(rex_base_url)
-    assert response.url.endswith(cnx_page_slug)
+    print()
```

Comment:
print statements are suppressed by the test framework unless you use `-s` or the test fails. Generally, printing should be avoided in tests. These print statements are also confirmed by your assert statements. Please get rid of them. Additionally, the test framework is set to `showlocals` which will print out any local variables. if you need insight to why a test failed you can set a value to a variable.